docs/advanced/directories.rst

```diff
@@ -0,0 +1,28 @@
+.. _directories:
+
+Organizing cookiecutters in directories (1.7+)
+---------------------------------------------------
+
+*New in Cookiecutter 1.7*
+
+Cookiecutter introduces the ability to organize several templates in one
+repository or zip file, separating them by directories. This allows using
+symlinks for general files, to edit them in one place. Here is how the
+structure looks for repositories or archives with this feature enabled::
+
+    https://github.com/user/repo-name.git
+        ├── directory1-name/
+        |   ├── {{cookiecutter.project_slug}}/
+        |   └── cookiecutter.json
+        └── directory2-name/
+            ├── {{cookiecutter.project_slug}}/
+            └── cookiecutter.json
+
+To activate one of templates from subdirectory use cli with ``--directory`` option::
+
+    cookiecutter https://github.com/user/repo-name.git --directory="directory-name"
+
+
+``--directory`` is a simple "transformation", where cookiecutter looks for
```

Comment:
We could probably lose this whole section?
Although it does make me wonder what would happen if you had an existing repo with `cookiecutter.json` in the root?